index.js

```diff
@@ -58,6 +60,7 @@ function multidrive (store, createArchive, closeArchive, cb) {
   }
 
   function create (data, cb) {
+    if (_disconnected) return cb(new Error('disconnected'))
```

Comment:
those callbacks should be called after `setImmediate()`, a callback shouldn't be called in the same tick as its function is invoked

Reply:
I thought so too but on other places it also immediately returned _(when in rome...)_. I took the liberty to fix all occurrences: https://github.com/datproject/multidrive/pull/14/commits/c621c0920856b42a867d697173cfaf4467b5523d